Server/src/bmp/BMPReader.cpp

```diff
@@ -114,9 +114,9 @@ bool BMPReader::ReadIncomingMsg(BMPListener::ClientInfo *client, MsgBusInterface
     char bmp_type = 0;
 
     MsgBusInterface::obj_router r_object;
-    memcpy(router_hash_id, client->hash_id, sizeof(router_hash_id));    // Cache the router hash ID (hash is generated by BMPListener)
+//    memcpy(router_hash_id, client->hash_id, sizeof(router_hash_id));    // Cache the router hash ID (hash is generated by BMPListener)
```

Comment:
Need to initially hash using hash type 0 (c_ip) because we may not get the router init as the first message. 

Need to also persist the hash since r_object is init each read.